app.py

```diff
@@ -1,9 +1,10 @@
 import os, requests, json
 
 from flask import Flask, render_template, request, flash, redirect, session, g 
+from flask_mail import Mail, Message
```

Comment:
let's make sure this works on Render. You might have to use an email service of some kind.